benchmarks/synchrobench/src/main/java/com/yahoo/oak/Memcached.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2020, Verizon Media.
+ * Licensed under the terms of the Apache 2.0 license.
+ * Please see LICENSE file in the project root for terms.
+ */
+
+package com.yahoo.oak;
+
+import com.yahoo.oak.synchrobench.contention.abstractions.BenchKey;
+import com.yahoo.oak.synchrobench.contention.abstractions.BenchValue;
+import com.yahoo.oak.synchrobench.contention.abstractions.KeyGenerator;
+import com.yahoo.oak.synchrobench.contention.abstractions.ValueGenerator;
+import com.yahoo.oak.synchrobench.contention.benchmark.Parameters;
+import com.yahoo.oak.synchrobench.maps.BenchMap;
+import net.spy.memcached.AddrUtil;
+import net.spy.memcached.ConnectionFactoryBuilder;
+import net.spy.memcached.MemcachedClient;
+import org.openjdk.jmh.infra.Blackhole;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Required installation of Memcached on the machine.
+ */
+public class Memcached extends BenchMap {
```

Comment:
Where exactly this benchmark is used? I mean in `run.h` I do not see it as part of benchmarks, while chronicle is there. Is it for future use? Why not to add a possibility to run.h?

Reply:
I will add it to `run.sh`, but I can't include it in the unit testing because it will require starting a Memcached service.
The reason I didn't add it was because the performance was not even at the same scale as the others.
But it might have been a configuration issue.